pennylane_lightning/src/util/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 project(lightning_utils LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
 
-add_library(lightning_utils INTERFACE)
+set(UTIL_FILES RuntimeInfo.cpp CACHE INTERNAL "" FORCE)
```

Comment:
I couldn't get really. Would you elaborate on this?

Reply:
This is because `util` now builds `RuntimeInfo.cpp`. So it is not an interface library anymore.